.gitignore

```diff
@@ -0,0 +1 @@
+os2mo_tools/__pycache__/
```

Comment:
You’ll probably also want to ignore the distutils `build` directory.

Reply:
Added.